com.jeffser.Alpaca.Plugins.AMD.json

```
@@ -0,0 +1,35 @@
+{
+    "id": "com.jeffser.Alpaca.Plugins.AMD",
+    "branch": "1.0",
+    "runtime": "com.jeffser.Alpaca",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//46",
+    "build-extension": true,
+    "cleanup": [
+        "*.a",
+        "*.la"
+    ],
+    "modules": [
+        {
+            "name": "rocm",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -r --remove-destination * ${FLATPAK_DEST}"
+            ],
+            "post-install": [
+                "install -Dm0644 com.jeffser.Alpaca.Plugins.AMD.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
```

```suggestion
            "post-install": [
                "rm ${FLATPAK_DEST}/com.jeffser.Alpaca.Plugins.AMD.metainfo.xml",
                "install -Dm0644 com.jeffser.Alpaca.Plugins.AMD.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
```

It's getting copied twice due to `cp *`